product_docs/docs/epas/15/epas_compat_ora_dev_guide/index.mdx

```diff
@@ -10,19 +10,19 @@ legacyRedirectsGenerated:
 
 <div id="introduction" class="registered_link"></div>
 
-Database compatibility for Oracle means that an application runs in an Oracle environment as well as in the EDB Postgres Advanced Server environment with minimal or no changes to the application code. Developing an application that's compatible with Oracle databases in the EDB Postgres Advanced Server requires special attention to the features used when constructing the application. For example, developing a compatible application means choosing compatible:
+EDB makes Postgres look, feel, and operate more like Oracle, so when you migrate, your developers have less code to rewrite and your DBAs can hit the ground running. Developing an application that's compatible with Oracle databases in the EDB Postgres Advanced Server requires special attention to the features used when constructing the application. For example, developing a compatible application means choosing compatible:
 
 -   System and built-in functions for use in SQL statements and procedural logic
 -   Stored procedure language (SPL) when creating database server-side application logic for stored procedures, functions, triggers, and packages
 -   Data types that are compatible with Oracle databases
 -   SQL statements that are compatible with Oracle SQL
 -   System catalog views that are compatible with Oracle’s data dictionary
 
-For detailed information about the compatible SQL syntax, data types, and views, see [Database Compatibility for Oracle Developers SQL](/epas/latest/epas_compat_sql/).
+For detailed information about the compatible SQL syntax, data types, and views, see [Database Compatibility for Oracle Developers SQL](../epas_compat_sql/).
 
-For the compatibility offered by the procedures and functions that are part of the built-in packages, see [Database Compatibility for Oracle Developers Built-in Packages](/epas/latest/epas_compat_bip_guide/).
+For the compatibility offered by the procedures and functions that are part of the built-in packages, see [Database Compatibility for Oracle Developers Built-in Packages](../epas_compat_bip_guide/).
 
-For information about using the compatible tools and utilities (EDB\*Plus, EDB\*Loader, DRITA, and EDB\*Wrap) that are included with an EDB Postgres Advanced Server installation, see [Tools and utilities](/epas/latest/epas_compat_ora_dev_guide/09_tools_and_utilities/).
+For information about using the compatible tools and utilities (EDB\*Plus, EDB\*Loader, DRITA, and EDB\*Wrap) that are included with an EDB Postgres Advanced Server installation, see [Tools and utilities](../epas_compat_ora_dev_guide/09_tools_and_utilities/).
 
 For applications written using the Oracle Call Interface (OCI), EDB’s Open Client Library (OCL) provides interoperability with these applications. For detailed information about using the Open Client Library, see [EDB OCL Connector](/ocl_connector/latest/).
```

Comment:
The second sentence can also be written as "See [EDB OCL Connector](/ocl_connector/latest/) for information about using the Open Client Library." Both the sentence are starting with 'For', so it's sounding a bit repetitive. Just a suggestion.